pkgs/test/cuda/cuda-samples/generic.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchFromGitHub
+, pkg-config
+, sha256, cudatoolkit
+}:
+
+let
+  pname = "cuda-samples";
+  version = lib.versions.majorMinor cudatoolkit.version;
+in
+
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = pname;
+    rev = "v${version}";
+    inherit sha256;
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  propagatedBuildInputs = [ cudatoolkit ];
```

Comment:
Why propagated? Shouldn't that be a regular nativeBuildInput?

Reply:
Yes I suppose it is just exes.